xcell/mappers/mapper_ACTk.py

```diff
@@ -5,19 +5,32 @@
 
 
 class MapperACTk(MapperACTBase):
+    """
+    For X either 'BN' or 'D56' depending on the desired sky patch.
```

Comment:
Same as above. Explain this better

Reply:
I see it happens also in other classes, I will not repeat it. So fix this in all of them